### DISCUSSION TITLE
Domains: Invalid configuration with correct config setup?

### DISCUSSION THREAD

**Discussion by jordanpunzalann** [vercel/community#4602] [Help] [answered]
### Summary

Hi, just trying to register the blitmap.comic domain to my website https://blitmap-comic-gallery.vercel.app/. I've configured the settings correct (screenshots below), and it has been 48+ hours with the dashboard still saying it's incorrectly configured. How should I proceed in making it work?

### Steps to Reproduce

Vercel configurations: 
![CleanShot 2023-10-25 at 13 24 10@2x](https://github.com/vercel/community/assets/8903600/ab32e1dd-a164-4986-98a0-3b03863302d7)

DNS configurations (namecheap):
<img width="1149" alt="CleanShot 2023-10-25 at 13 24 14@2x" src="https://github.com/vercel/community/assets/8903600/75a7d2f4-cd26-4594-a044-79e512b3972b">

**Comment by amyegan** [comment, marked as answer]
Hi @jordanpunzalann. A `.comic` domain is a handshake domain, which is based on a decentralized blockchain-system and not compatible with most existing DNS systems. 

You can see this by checking the [A record for blitmap.comic](https://www.whatsmydns.net/#A/blitmap.comic) and the [CNAME record for www.blitmap.comic](https://www.whatsmydns.net/#CNAME/www.blitmap.comic). From the perspective of traditional DNS systems, the records are legitimately missing. As a result, the domain will not resolve in regular browsers. 

Further setup is needed to work with HNS domains. Please ask your domain registrar for help with this. You can learn more about Namecheap Handshake domains at https://www.namecheap.com/support/knowledgebase/article.aspx/10484/2278/namecheap-handshake-tlds/
